ui/src/components/Treasury.jsx

```diff
@@ -79,11 +102,15 @@ function VaultList() {
     );
   }
 
+  if (loadTreasuryError) {
+    return loadTreasuryErrorAlert;
+  }
+
   if (vaults === null) {
     return <CircularProgress style={{ marginTop: 48 }} />;
   }
 
-  if (vaultsList.length <= 0) {
+  if (vaultsList.length === 0) {
```

Comment:
Is this a substantive change or just style?

Reply:
Just style

Reply:
just style, as we discussed